### PR TITLE
[Docs] Add built-in observability dashboard

### PR DESCRIPTION
**Pull request by justindeguzman**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add details about built-in observability dashboard

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit 747deb13e2d9b96d5ba30168f9e3423a6c4b1e01 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56647/747deb13e2d9b96d5ba30168f9e3423a6c4b1e01/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6844839741">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56647/747deb13e2d9b96d5ba30168f9e3423a6c4b1e01/ci_running.html">❌ failure</a></td></tr>
<tbody>
</table>

**Review by alexey-milovidov — APPROVED** [review]

File: docs/en/operations/monitoring.md [hunk 1]
```diff
@@ -15,6 +15,27 @@ You can monitor:
 - Utilization of hardware resources.
 - ClickHouse server metrics.
 
+## Built-in observability dashboard
+
+<img width="400" alt="Screenshot 2023-11-12 at 6 08 58 PM" src="https://github.com/ClickHouse/ClickHouse/assets/3936029/2bd10011-4a47-4b94-b836-d44557c7fdc1">
+
+ClickHouse comes with a built-in observability dashboard feature which can be accessed by `$HOST:$PORT/dashboard` (requires user and password) that shows the following metrics:
```
Comment (johnnymatthews) [inline docs/en/operations/monitoring.md:22]:
```suggestion
ClickHouse comes with a built-in observability dashboard feature which can be accessed by `$HOST:$PORT/dashboard` (requires username and password) that shows the following metrics:
```

Comment (johnnymatthews) [reply docs/en/operations/monitoring.md:22]: Should be _username_?

**Review by johnnymatthews — CHANGES_REQUESTED** [review]

File: docs/en/operations/monitoring.md (hunk 1 above)
Comment (justindeguzman) [reply docs/en/operations/monitoring.md:22]: It's a database `user`: https://clickhouse.com/docs/en/sql-reference/statements/create/user. The dashboard UI also references `user`

**Review by johnnymatthews — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@justindeguzman tests have failed.

**Comment by alexey-milovidov** [comment]
@johnnymatthews tests have failed.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**